documentation/release_notes/release_notes_latest.markdown

```diff
@@ -20,6 +20,34 @@ canonical: "/puppetdb/latest/release_notes.html"
 [puppet_apply]: ./connect_puppet_apply.html
 
 ---
+
+## PuppetDB 6.10.0
+
+### Upgrading
+
+This upgrade contains a very long running migration to the reports table,
+which is typically the largest table in PuppetDB by far. Before upgrading to,
+or past, this version of PuppetDB, you are strongly encouraged to consider
+deleting your reports table. This will drastically shorten your upgrade time
+and get you back online much faster. If you are on a version `5.2.z`, please
+upgrade to `5.2.14` or later and then take advantage of the `delete-reports`
+subcommand. Otherwise, consult the documentation on how to [truncate the
+reports table manually](https://puppet.com/docs/puppetdb/latest/upgrade.html#truncate-your-reports-table).
```

Comment:
```suggestion
This upgrade contains a long running migration to the reports table,
which is typically the largest table in PuppetDB. Before upgrading to,
or past, this version of PuppetDB, you are strongly encouraged to consider
deleting your reports table. This will drastically shorten your upgrade time
and get you back online much faster. If you are on a `5.2.z` version, please
upgrade to `5.2.14` or later and then take advantage of the `delete-reports`
subcommand. Otherwise, consult the documentation on how to [truncate the
reports table manually](https://puppet.com/docs/puppetdb/latest/upgrade.html#truncate-your-reports-table).
```

Reply:
I'm assuming the "truncate instructions" in the docs haven't hit master yet?

Reply:
Correct, they are in a PR,  puppetlabs/puppetdb#3239